packaging/linux/deb/debroot.go

```diff
@@ -277,6 +278,26 @@ func fixupArtifactPerms(spec *dalec.Spec, target string, cfg *SourcePkgConfig) [
 		}
 	}
 
+	if len(artifacts.Links) > 0 {
+		fmt.Fprintf(buf, "# Set ownership for artifact symlinks\n")
+		for _, link := range artifacts.Links {
+			if link.UID != 0 || link.GID != 0 {
+				fmt.Fprintf(buf, "chown -h %d:%d \"$DESTDIR%s\"\n", link.UID, link.GID, link.Dest)
+			}
+		}
+	}
+
+	if spec.Image != nil && spec.Image.Post != nil {
```

Comment:
This doesn't belong here.
The image post stuff is not part of the packaging and is instead used to modify the container after it is built.